lib/common/grpc-service.js

```diff
@@ -266,6 +266,25 @@ GrpcService.prototype.request = function(protoOpts, reqOpts, callback) {
     request: function(_, onResponse) {
       respError = null;
 
+      if (stream) {
```

Comment:
[`Table#getRows`](https://github.com/callmehiphop/gcloud-node/blob/bigtable/lib/bigtable/table.js#L421) and [`Table#sampleRowKeys`](https://github.com/callmehiphop/gcloud-node/blob/bigtable/lib/bigtable/table.js#L621)